types/artifacts.d.ts

```diff
@@ -115,7 +115,7 @@ declare global {
       /** All the link elements on the page or equivalently declared in `Link` headers. @see https://html.spec.whatwg.org/multipage/links.html */
       LinkElements: Artifacts.LinkElement[];
       /** The values of the <meta> elements in the head. */
-      MetaElements: Array<{name?: string, content?: string, property?: string, httpEquiv?: string, charset?: string}>;
+      MetaElements: Array<{name?: string, content?: string, property?: string, httpEquiv?: string, charset?: string, node: LH.Artifacts.NodeDetails}>;
```

Comment:
This change introduces a type error on a seemingly unrelated file (`stacks.js` snapshot fn)

Reply:
I'm having a hard time finding a link as well. Could this be a TS bug?

I was able to fix the error by making `node` optional though.

Reply:
https://github.com/GoogleChrome/lighthouse/pull/12280